.storybook/pages/WireframeDemo/GlobalStyles.module.css

```diff
@@ -11,6 +11,9 @@
  */
 .root {
   /* Level 1 wireframe theme tokens */
+  --wireframe-anim-duration: 0;
+  --wireframe-font-family-primary: 'Balsamiq Sans', sans-serif;
+  --wireframe-font-family-secondary: 'Redacted Script', sans-serif;
```

Comment:
I added these so all the overrides are using a wireframe token or `transparent` (which I guess I could add a token for if we wanted but eh).